dependencies.rosinstall

```diff
@@ -1,8 +1,8 @@
 - git:
     local-name: opw_kinematics
     uri: https://github.com/Jmeyer1292/opw_kinematics.git
-    version: master
+    version: 0.3.1
```

Comment:
We should probably get rid of OPW kinematics in the rosinstall file since this repository won't depend on it any more

Reply:
Good catch.

Reply:
It has been removed.